### DISCUSSION TITLE
How to reset to default?

### DISCUSSION THREAD

**Discussion by sdutky** [AppGround-io/bluetooth-keyboard-and-mouse-support#269] [Q&A]
Uninstall then reinstall?  Does this keep my premium subscription?

**Comment by AdamKane41** [comment]
Yes you will keep premium.
But to reset the app to default you can also, [Force Stop] the app via Android's [App Info] page and Clear/Delete the app's [Cache] AND [Data]. 🙂

**Comment by sdutky** [comment]
Thanks!

On Thu, Nov 17, 2022 at 3:56 PM [AdamKane] ***@***.***> wrote:

> Yes you will keep premium.
> But to reset the app to default you can also, [Force Stop] the app via
> Android's [App Info] page and Clear/Delete the app's [Cache] AND [Data].
> 🙂
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/AppGround-io/bluetooth-keyboard-and-mouse-support/discussions/269#discussioncomment-4171431>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAZ6ESIXYSX2OMNDTETRBWDWI2LXRANCNFSM6AAAAAAR6OTPZQ>
> .
> You are receiving this because you authored the thread.Message ID:
> <AppGround-io/bluetooth-keyboard-and-mouse-support/repo-discussions/269/comments/4171431
> @github.com>
>


-- 

If they can get you asking the wrong questions, they don't have to worry
about answers.

–*Thomas Pynchon, Gravity's Rainbow*
